src/st/no3/3FF00.c

```diff
@@ -238,7 +238,325 @@ void EntityDeathStolenItem(Entity* self) {
     }
 }
 
-INCLUDE_ASM("asm/us/st/no3/nonmatchings/3FF00", EntityUnkId5A);
+const s32 rodata_padding_801B7510 = 0;
+
+void EntityDeath(Entity* self) {
+    Entity* newEntity = &self[1];
+    Primitive* prim;
+    s16 left, right;
+    s32 primIndex;
+    s32 x, y;
+    s32 i;
+
+    if ((self->step >= 4) && (self->step < 13)) {
+        if (self->ext.death.unk84 != 0) {
```

Comment:
To me it looks like this is what is making him move up and down. 

unk84 -> moveTimer
unk86 -> moveDirection

Names are just suggestions, call it whatever